drafts/one-percent-for-open-source.rst

```diff
@@ -0,0 +1,111 @@
+* **Startups** invest in the tools that they use, to *ensure continued support and maintenance*, to remove risk from their future, and have better tools to build their company with. It will also make *hiring developers* much easier for the companies, because developers love companies that invest in open source.
+* **Open source projects** are able to *fund project management and maintenance* that is required to make them sustainable for the long term. This continues the *force multiplier* that open source has given to the software industry as a whole.
+
+Why Venture Capital payments?
+-----------------------------
+
+The biggest reason is because all parties involved benefit from open source.
+Secondly,
+having a concrete point in time also makes it much easier to define when the money should be invested.
+Third,
+a set amount of money is already changing hands during an investment,
+so having 1% that money to go open source is easy.
+
+I also think monetary investment is required to make a real dent in this problem. 
+A lot of places have things like "Open Source Fridays" and "20% time",
+but those programs are the first to be cut when a company has issues.
+Investing in open source as a lump sum during funding rounds makes sure the investment up front,
+where it can't be removed as an ongoing program.
+
+How should we implement it?
+---------------------------
+
+The startup who has gotten the investment should invest 1% of the money from the VC firm into the open source projects that they use.
+
+There are many different ways this could be implemented as a lower level.
+A few of my ideas are:
+
+* Startup companies create a list of the OSS projects they use, using tools like http://libraries.io/, and then give money to the 10 largest projects that have a way to accept money. This could be done during the license compliance phase of the investment process.
+* There could be a new non-profit foundation created, which gets all the donations and then has established criteria for how to reinvest the money as grants.
+* You could give the money to the existing foundations that focus on the area of open source that is most valuable to you 
+
+I think making the execution of the investment standard is an important part of making this actually happen.
+There should be standard processes and criteria for where your money will go,
+so that it's not adding additional overhead to your funding round.
+
+A simple example for a Python project could be:
+
+* You give 1% of your Venture Capital investment to the Python Software Foundation
+* You run a script that looks at all the projects in your `requirements.txt` that your project depends on
+* That list is cross-referenced with a list of the Python projects you use that need funding
+* The top 10 projects you use that need funding, sorted by amount of funding needed, are each given 10% of your investment as a sustainability grant
```

Comment:
This approach would mean Django got really well funded, but django-niche-middleware-plugin wouldn't. The long tail is where a lot of the value is, but it's in just as much need of maintenance.

Reply:
Although Django could equally redistribute what it gets, no?

Reply:
My thought here is that it would be given in terms of "money needed that they don't have", so that it would naturally trickle down. It's just a simple example of a system, not the proposed best solution.

Reply:
sorting by "amount of money invested so far that year" lowest first reverses this process, I'll go with that.